zeppelin-interpreter/src/main/thrift/RemoteInterpreterService.thrift

```diff
@@ -56,6 +56,11 @@ struct RemoteInterpreterEvent {
   2: string data      // json serialized data
 }
 
+struct InterpreterCompletion {
+  1: string name,
+  2: string value
```

Comment:
They PR is huge so sorry if I have missed it, but could you please point to the place where this API is used?
Right now, looking at this it is not clear what exactly is `name` and `value` here